content/ios/user.md

```diff
@@ -79,11 +91,15 @@ if (container.currentUserRecordID) {
 
 ## Logging out
 
+Of course, you also would want to allow users to log out of their accounts by using `logoutWithCompletionHandler:`.
```

Comment:
I actually copied this from Parse. I think it would make the documentation less monotonic since I think Skygear is targeted towards not-really-technical developers.